src/plugins/intel_cpu/src/nodes/subgraph.cpp

```diff
@@ -519,7 +519,11 @@ Snippet::SnippetJitExecutor::SnippetJitExecutor(const SnippetAttrs& attrs, bool
         snippet_for_generation->set_friendly_name(snippetAttrs.snippet->get_friendly_name());
         auto host_isa = dnnl::impl::cpu::x64::mayiuse(dnnl::impl::cpu::x64::avx512_core) ?
             dnnl::impl::cpu::x64::avx512_core : dnnl::impl::cpu::x64::avx2;
+#if defined(OPENVINO_ARCH_X86_64)
```

Comment:
why is it Windows arm64 specific?

Reply:
CPUGenerator is excluded on ARM64 by the code (https://github.com/openvinotoolkit/openvino/blob/2cd130810432662f9974521e76418d876ef470e1/src/plugins/intel_cpu/CMakeLists.txt#L81)